CHANGELOG.md

```diff
@@ -82,6 +82,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Added cluster manager throttling stats in nodes/stats API ([#5790](https://github.com/opensearch-project/OpenSearch/pull/5790))
 - Added support for feature flags in opensearch.yml ([#4959](https://github.com/opensearch-project/OpenSearch/pull/4959))
 - Add query for initialized extensions ([#5658](https://github.com/opensearch-project/OpenSearch/pull/5658))
+- Fix flaky test cases ([#5952](https://github.com/opensearch-project/OpenSearch/pull/5952))
```

Comment:
Purely test related changes do not need a changelog entry. See [CONTRIBUTING.md#changelog](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#changelog).

Reply:
Removed the CHANGELOG.md file.